src/TRestRawSignalEvent.cxx

```diff
@@ -501,7 +501,7 @@ TPad* TRestRawSignalEvent::DrawEvent(TString option) {
 
     ///// No specific signal selection ////
     if ((optList.size() == 0) || (isANumber((string)optList[0]) == false)) {
-        if (mg != NULL) delete mg;
+        if (mg) delete mg;
```

Comment:
same

Reply:
Well it should actually be

`delete mg;`, since nothing happens if you delete a null pointer, unless I am missing something, I will end up writing a python script to replace all these statements.

Reply:
Fair enough, yeah.